pkg/helper/helper.go

```diff
@@ -0,0 +1,251 @@
+package helper
```

Comment:
Did you try adding operator as a  dependency?

Reply:
Yes, I have used ```github.com/tektoncd/operator/pkg/apis/operator/v1alpha1``` config schema to validated cluster Resource Creation & Deletion